docs/team/weijie96.md

```diff
@@ -12,26 +12,45 @@ It is written in Java, and has about 10 kLoC.
 Given below are my contributions to the project.
 
 * **New Feature**: Added the ability to archive clients.
-  * What it does: **Coming Soon**
-  * Justification: **Coming Soon**
-  * Highlights: **Coming Soon**
+  * What it does: It allows the user to add active clients into the archive. The opposite can be done as well; archived clients can be made active again.
+  To support the archiving feature, users can also view the archive, as well as to switch back to view the active clients.
+  * Justification: This feature improves the product significantly because a user can add clients which are not currently relevant to the user into the archive.
+  The presence of a separate view of the archive prevents the user from getting distracted by archived clients. This is different from deleting a client,
+  as it handles for the case when the archived client turning active again in future.
+  * Highlights: 
+  This enhancement affects existing commands (such as `ListCommand` and `AddCommand`), and commands to be added in future (such as `FindCommand`). 
+  It required an in-depth analysis of design alternatives. 
+  The implementation too was challenging as it required changes to existing commands.
```

Comment:
Just a suggestion, maybe you can elaborate on how is it challenging, or highlight one or two cases that were really difficult for you. Or you can edit the DG, then link it there also.

Reply:
Updated